content/css/concepts/filter-functions/terms/contrast/contrast.md

```diff
@@ -38,3 +38,6 @@ Give the image a contrast of `200%`:
   filter: contrast(200%);
 }
 ```
+![Contrast example](/media/css-filterfunctions-contrast-example.png)
```

Comment:
Add a more descriptive `alt-text` (in the square brackets). Something like this:
Comparison between the image with contrast to the original image